tools/run_tests/run_interop_tests.py

```diff
@@ -535,10 +535,10 @@ def build_interop_image_jobspec(language, tag=None):
     env['TTY_FLAG'] = '-t'
   # This env variable is used to get around the github rate limit
   # error when running the PHP `composer install` command
-  # TODO(stanleycheung): find a more elegant way to do this
-  if language.safename == 'php' and os.path.exists('/var/local/.composer/auth.json'):
+  host_file = '%s/.composer/auth.json' % os.environ['HOME']
```

Comment:
I think python has a function to tell you the home dir of current user, which would be more portable, but fair enough.